include/boost/math/distributions/inverse_gamma.hpp

```diff
@@ -195,6 +196,43 @@ inline RealType pdf(const inverse_gamma_distribution<RealType, Policy>& dist, co
    return result;
 } // pdf
 
+template <class RealType, class Policy>
+inline RealType logpdf(const inverse_gamma_distribution<RealType, Policy>& dist, const RealType& x)
+{
+   BOOST_MATH_STD_USING  // for ADL of std functions
+   using boost::math::tgamma;
+
+   static const char* function = "boost::math::logpdf(const inverse_gamma_distribution<%1%>&, %1%)";
+
+   RealType shape = dist.shape();
+   RealType scale = dist.scale();
+
+   RealType result = 0;
```

Comment:
-INF for all the special cases I *think*.  Need to change the `return 0;` to `return result;` as well.